garak/detectors/fileformats.py

```diff
@@ -0,0 +1,90 @@
+# SPDX-FileCopyrightText: Portions Copyright (c) 2024 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+"""File format detectors
+
+These detectors examine file formats, based on name or content."""
+
+import pickletools
+
+import magic
+
+from garak.detectors.base import FileDetector
+
+
+class PossiblePickleName(FileDetector):
```

Comment:
Grumble grumble filename-based detections etc etc

Reply:
Would you rather we skipped this entirely? I'm not married to it